pkg/azure/plugin.go

```diff
@@ -90,7 +89,8 @@ func (s *azurePluginServer) GetPermitList(ctx context.Context, req *paragliderpb
 
 	// get the NSG rules
 	for _, rule := range nsg.Properties.SecurityRules {
-		if !strings.HasPrefix(*rule.Name, denyAllNsgRulePrefix) && strings.HasPrefix(*rule.Name, paragliderPrefix) {
+		// Ignore default Azure rules with priority above maxPriority
+		if *rule.Properties.Access == allowRule && *rule.Properties.Priority <= maxPriority {
```

Comment:
There shouldn't be any deny rules, right? So is the first part of this check even necessary?

Reply:
There's the deny-all rule for inbound and outbound rules

Reply:
Removing the <= maxPriority because Rules, doesn't contain the default Azure rules, so no need for that